src/renderers/dom/shared/__tests__/ReactDOMComponent-test.js

```diff
@@ -618,6 +618,22 @@ describe('ReactDOMComponent', function() {
       expect(SimpleEventPlugin.didPutListener.mock.calls.length).toBe(2);
       expect(SimpleEventPlugin.willDeleteListener.mock.calls.length).toBe(1);
     });
+
+    it('should warn for children on void elements', function() {
+      spyOn(console, 'error');
+      var X = React.createClass({
+        render: function() {
+          return <input>moo</input>;
+        }
```

Comment:
lint wants a trailing comma here… not really my favorite style but we did it so let's stick with it and maybe revisit that dangle rule later.